synapse/replication/tcp/redis.py

```diff
@@ -328,6 +328,6 @@ def lazyConnection(
     factory.continueTrying = reconnect
 
     reactor = hs.get_reactor()
-    reactor.connectTCP(host, port, factory, 30)
+    reactor.connectTCP(host, port, factory, timeout=30, bindAddress=None)
```

Comment:
mypy was complaining about a missing `bindAddress` parameter, this gives the default value.

Reply:
(This is because the interface doesn't define the default, but the implementations do.)